src/site/_includes/js/eligibility.js

```diff
@@ -664,7 +664,17 @@ function addListeners() {
   document.getElementById("next-button").addEventListener("click", toNextPage);
   document.getElementById("back-button").addEventListener("click", toPrevPage);
   document.getElementById("submit-button").addEventListener("click", submitForm);
-
+  document.addEventListener("keypress", function(event) {
+    if ( event.key === "Enter" && ( document.activeElement.tagName === "INPUT" || document.activeElement.tagName === "SELECT" ) ){
```

Comment:
We generally try to follow the Google JS style guide, which recommends omitting spaces just inside the parentheses.

```
if (thisIsGood) {

if ( notRecommended ) {
```

https://google.github.io/styleguide/jsguide.html#formatting-horizontal-whitespace